src/structures/MessageMentions.js

```diff
@@ -125,6 +125,12 @@ class MessageMentions {
     } else {
       this.crosspostedChannels = new Collection();
     }
+
+    /**
+     * The author of the message that this message is a reply to
+     * @type {?User}
+     */
+    this.repliedUser = repliedUser ? this.client.users.add(repliedUser) : this.repliedUser ?? null;
```

Comment:
```suggestion
    this.repliedUser = repliedUser ? this.client.users.add(repliedUser) : null;
```
thats never assigned before so its just a useless check

Reply:
It protects it against message edits which might not include that property though.

Reply:
this assignment is in the constructor for `MessageMentions`, so `this.repliedUser ?? null` would always just return null here no? so why not just directly set it to null